examples/nifgen_standard_function/_nifgen_helpers.py

```diff
@@ -0,0 +1,40 @@
+"""nifgen Helper classes and functions for MeasurementLink examples."""
+
+from typing import Any, Dict, Optional
+
+import grpc
+import nifgen
+
+import ni_measurementlink_service as nims
+
+# To use a physical NI FGen instrument, set this to False or specify
+# --no-use-simulation on the command line.
+USE_SIMULATION = True
+
+
+def _create_nifgen_session(
+    session_info: nims.session_management.SessionInformation,
+    session_grpc_channel: grpc.Channel,
+    initialization_behavior: Optional[
+        nifgen.SessionInitializationBehavior
+    ] = nifgen.SessionInitializationBehavior.AUTO,
```

Comment:
Optional parameters don't have to have a type of Optional[T]. Optional[T] means nullable (can be None) which is different than optional as in "has a default". 

You can remove the type annotation for this parameter and let mypy infer it from the default value.

Did we get this wrong in the DCPower example too?